include/globals.h

```diff
@@ -1440,15 +1436,13 @@ extern RemoteDebug Debug;           // Let everyone in the project know about it
 
 #if USE_SCREEN
 
-    #if ARDUINO_HELTEC_WIFI_KIT_32_V3
-
-        #define USE_OLED 1
-        #define USE_SSD1306 1
-
-    #elif ARDUINO_HELTEC_WIFI_KIT_32
+    #if ARDUINO_HELTEC_WIFI_KIT_32
                         // screen definations for heltec_wifi_kit_32 or heltec_wifi_kit_32_v2
 
         #define USE_OLED 1                                    // Enable the Heltec's monochrome OLED
+        #ifndef USE_SSD1306
```

Comment:
`USE_SSD1306` is a flag that can also be defined to 0, in which case the line below should not be included. I therefore think the `#ifndef` should be replaced, as follows:

```suggestion
        #if !(USE_SSD1306)
```

Reply:
Makes sense to me. I tested this change locally. Everything still worked as expected, and I verified omitting the "-DUSE_SSD1306=1" flag from the environment in platformio.ini will trigger this conditional, as expected.